DX/Docker/Compose.md

```diff
@@ -0,0 +1,35 @@
+# v2: Docker Compose Requirements
+
+## Objective
+To provide a default Docker Compose file or files, ready to use out-of-the-box.
```

Comment:
It's not just docker compose, blue print or whatever we call it also involves docker files (containers), and ways to configure eZ Platform from docker (env variables) which needs changes on eZ Platform itself _(like in 2.0 branch: https://github.com/ezsystems/ezplatform/pull/167/files#diff-bf0e70bcef1a5d5b2f87289220a51108R10)_